app/main/views/platform_admin.py

```diff
@@ -771,3 +781,76 @@ def _get_user_row(r):
     row.append(r["password_changed_at"])
     row.append(r["state"])
     return row
+
+
+@main.route(
+    "/platform-admin/load-test",
+    methods=["POST", "GET"],
+)
+@user_is_platform_admin
+def load_test():
+    """
+    The load test assumes that a service called 'Test service' exists.  It will make
+    the platform admin a member of this service if the platform is not already. All
+    messagese will be sent in this service.
+    """
+    # SIMULATED_SMS_NUMBERS = ("+14254147755", "+14254147167")
```

Comment:
We have these numbers defined as a global config for the main Flask app too, right?